src/rustdoc_schema.graphql

```diff
@@ -517,11 +517,62 @@ A specific attribute applied to an Item.
 """
 type Attribute {
   """
-  The textual representation of the attribute.
+  String representation of the attribute as it is found in the code.
 
-  For example: "#[repr(C)]"
+  For example: `#[non_exhaustive]`
   """
-  value: String!
+  as_string: String!
```

Comment:
What do you think about renaming `as_string` to something like `raw_attribute`, analogously to how `raw_item` on `AttributeMetaItem` is the raw string representation for that type?

Reply:
Good idea, I changed it to `raw_attribute`.